project/src/components/movie-page-reviews/movie-page-reviews.tsx

```diff
@@ -17,98 +19,20 @@ function MoviePageReviews({films}: MoviePageReviewsProps): JSX.Element {
-
-          <div className="review__rating">8,0</div>
-        </div>
-
-        <div className="review">
-          <blockquote className="review__quote">
-            <p className="review__text">I did not find it amusing, and while I can appreciate the creativity,
-              it is an hour and 40 minutes I wish I could take back.
-            </p>
-
-            <footer className="review__details">
-              <cite className="review__author">Amanda Greever</cite>
-              <time className="review__date" dateTime="2015-11-18">November 18, 2015</time>
-            </footer>
-          </blockquote>
-
-          <div className="review__rating">8,0</div>
-        </div>
-      </div>
-      <div className="film-card__reviews-col">
-        <div className="review">
-          <blockquote className="review__quote">
-            <p className="review__text">The mannered, madcap proceedings are often delightful, occasionally
-              silly, and here and there, gruesome and/or heartbreaking.
-            </p>
-
-            <footer className="review__details">
-              <cite className="review__author">Matthew Lickona</cite>
-              <time className="review__date" dateTime="2016-12-20">December 20, 2016</time>
-            </footer>
-          </blockquote>
-
-          <div className="review__rating">7,2</div>
-        </div>
-
-        <div className="review">
-          <blockquote className="review__quote">
-            <p className="review__text">It is certainly a magical and childlike way of storytelling, even if
-              the content is a little more adult.
-            </p>
-
-            <footer className="review__details">
-              <cite className="review__author">Paula Fleri-Soler</cite>
-              <time className="review__date" dateTime="2016-12-20">December 20, 2016</time>
-            </footer>
-          </blockquote>
-
-          <div className="review__rating">7,6</div>
-        </div>
-
-        <div className="review">
-          <blockquote className="review__quote">
-            <p className="review__text">It is certainly a magical and childlike way of storytelling, even if
-              the content is a little more adult.
-            </p>
-
-            <footer className="review__details">
-              <cite className="review__author">Paula Fleri-Soler</cite>
-              <time className="review__date" dateTime="2016-12-20">December 20, 2016</time>
-            </footer>
-          </blockquote>
-
-          <div className="review__rating">7,0</div>
-        </div>
+        {reviews.map((elem) => (
+          <div className="review" key={elem.id}>
+            <blockquote className="review__quote">
+              <p className="review__text">{elem.text}</p>
+
+              <footer className="review__details">
+                <cite className="review__author">{elem.author}</cite>
+                <time className="review__date" dateTime="2016-12-24">{elem.date}</time>
```

Comment:
аттрибут `dateTime`тоже должен выводиться из моков согласно его формату

```suggestion
                <time className="review__date" dateTime="{???}">{elem.date}</time>
```